fs/glob.py

```diff
@@ -27,6 +26,54 @@
 )  # type: LRUCache[Tuple[Text, bool], Tuple[int, bool, Pattern]]
 
 
+def _translate(pattern, case_sensitive=True):
```

Comment:
Yes, pretty much, but not "moved" and rather "copied". 
I think it would be possible to just merge the two; at least, I don't see any difference in their feature set or behavior anymore. 

The only odd point I can find (from [here](https://docs.pyfilesystem.org/en/latest/globbing.html)) is:
> *.py?
> Matches all .py files and .pyi, .pyc etc in the currenct directory.

That's not the way [UNIX glob](https://man7.org/linux/man-pages/man7/glob.7.html) works (which this syntax is based on, I assume) or the way `fs.wildcard` works, nor is this actually tested anywhere. So it might just be an error in documentation.

Reply:
Yes, I think it's a documentation error too. The current implementation just replaces a `?` in the wildcard with a `.` in the corresponding regex.